### ISSUE TITLE
The "Open file location" button and its icon is not understandable when exiting/closing a document

### ISSUE THREAD

**Issue by Jerome-Herbinet** [ONLYOFFICE/onlyoffice-nextcloud#677]
Hi!
My colleagues and I (and most of our clients) agree about the close document button. Its wording and icon do not make sense and does not fit the UI/UX habits found in most software. 
Instead of "Open file location" and a folder icon with an arrow in the middle of it, the simpler one will be more efficient and understandable. 
A simple cross with "Close document" is needed. The fact that the user goes back to the Nextcloud Files file explorer afterwards seems logical and implicit to me (so no need to introduce this notion in the button). First, we are looking for closing button ... and accessing to Nextcloud Files is only secondary. 
The first time I used OnlyOffice, I went round and round trying to figure out how to leave (close) the document :-)
Finally, when we train our clients to use OnlyOffice, it never makes sense to say... when you want to close the document, click on "Open file location". It is not intuitive.
What do you think about it? It would be great if you could change this.
Thanks for reading 🙂
Jérôme

**Comment by rasos** [comment]
Indeed,  after upgrading to Nextcloud 31, now with ONLYOFFICE App Version 9.11.0, there is no more button top right to leave the document, and there is also no such option in the file menu. We have configured OO to open in the same tab. Then also the back button does not work to leave the document to see the file tree.
